core/src/main/java/org/stellar/anchor/util/MathHelper.java

```diff
@@ -1,12 +1,18 @@
 package org.stellar.anchor.util;
 
 import java.math.BigDecimal;
+import java.math.RoundingMode;
+import org.stellar.anchor.asset.AssetInfo;
 
 public class MathHelper {
   public static BigDecimal decimal(String value) {
     return new BigDecimal(value);
   }
 
+  public static BigDecimal decimal(String value, AssetInfo asset) {
+    return new BigDecimal(value).setScale(asset.getSignificantDecimals(), RoundingMode.HALF_UP);
```

Comment:
🤔  I think we should actually round HALF_DOWN.

Reply:
Use HALF_DOWN.